lgn-worker/src/main.rs

```diff
@@ -140,57 +135,42 @@ async fn main() -> anyhow::Result<()> {
         );
     }));
 
-    let last_task_processed =
-        AtomicU64::new(SystemTime::now().duration_since(UNIX_EPOCH)?.as_secs());
-
-    if let Err(err) = run(cli, mp2_requirement, last_task_processed).await {
+    if let Err(err) = run(cli).await {
         panic!("Worker exited due to an error: {err:?}")
     } else {
         Ok(())
     }
 }
```

Comment:
Main now only install the panic handler and loads the config. Everything else is in `run`.